backup-organization-policy/main.tf

```diff
@@ -59,6 +59,19 @@ data "aws_iam_policy_document" "organization_backup_policy" {
       variable = "organizations:PolicyType"
       values   = ["BACKUP_POLICY"]
     }
+  }
+   statement {
+    sid = "AllowBackupPoliciesTagModification"
+    effect = "Allow"
+    principals {
+        identifiers = [var.delegate_account_id]
+        type        = "AWS"
+    }
+    resources = ["*"]
+    actions = [
+      "organizations:TagResource",
+      "organizations:UntagResource"
+    ]
```

Comment:
@MuhammeedAlaa , can you run` terraform fmt `on this file and update the branch?, we have a failed check on the fmt in this file.

Reply:
@OlamideOl1 Fixed please check out!